plugin/http/httpstats/client.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package httpstats
+
+import (
+	"net/http"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+	"log"
+	"fmt"
+	"context"
+	"time"
+	"strconv"
+	"net/http/httptrace"
+)
+
+var (
+	Started           *stats.MeasureInt64
```

Comment:
Is there a reason we don't have the same measures/views with the grpcstats package?

Reply:
I am going to reconcile these with https://github.com/census-instrumentation/opencensus-specs/blob/master/http/stats.md